All project code/views/pages/logout.ejs

```diff
@@ -0,0 +1,10 @@
+<%- include ('../partials/header') %>
+<%- include ('../partials/menu') %>
+
+<main>
+    <div class="container">
+        <h1>You have successfully logged out!</h1>
+    </div>
+</main>
+
+<%- include ('../partials/footer') %>
```

Comment:
For example, you probably don't want to re-commit the logout.ejs file. This should already exist in master